src/Views/Form/Templates/Sequoia/views/receipt.php

```diff
@@ -4,22 +4,23 @@
 use function Give\Helpers\Form\Template\get as getTemplateOptions;
 use function Give\Helpers\Form\Template\Utils\Frontend\getPaymentId;
 use function give_get_gateway_admin_label as getGatewayLabel;
+use function give_get_payment_status as getDonationStatusLabel;
 use function give_currency_filter as filterCurrency;
 use function give_sanitize_amount as sanitizeAmount;
 use function give_do_email_tags as formatContent;
 use Give_Payment as Payment;
 
-/* @var Payment $payment */
-$payment = new Payment( getPaymentId() );
-$options = getTemplateOptions();
+/* @var Payment $donation */
+$donation = new Payment( getPaymentId() );
```

Comment:
I notice that you just replaced `$payment` with `$donation`. What's the reason? Your dealing with an instance of `Payment`, so isn't `$payment` a more appropriate name?

Reply:
@JasonTheAdams In Give, we do not use payment instead we use `donation`. payment term still exists because code was forked.